docs/usage/CUSTOMER_SYNC.md

```diff
@@ -35,7 +35,7 @@ Any reference that is not expanded will have its id in place and not replaced by
 resources on the target commercetools project and the library will issue an update/create an API request without reference
 resolution.
 
-     - When syncing from a source commercetools project, you can use [`mapToCustomerDrafts`](https://commercetools.github.io/commercetools-sync-java/v/2.3.0/com/commercetools/sync/customers/utils/CustomerReferenceResolutionUtils.html#mapToCustomerDrafts-java.util.List-)
```

Comment:
same here, conflict looks like fixed in the wrong way.

Reply:
ea0287825